# Implements SBP MSG_IMU_AUX

uint8 imu_type  # IMU type
int16 temp      # Raw IMU temperature
uint8 imu_conf  # IMU conÔ¨Åguration
